client/admin/components/program-levels/ProgramLevel/index.vue

```diff
@@ -48,8 +48,10 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+$ligth-blue: #3273dc;
```

Comment:
You mean li**gh**t? 😁

Reply:
Also name it according to usage not visual properties :bowtie: (take this as rule of thumb when working with sass)